### DISCUSSION TITLE
miniMap for Leaftet

### DISCUSSION THREAD

**Discussion by toufa123** [TerriaJS/terriajs#5875] [Q&A]
I would like to add miniMap for Leaftet does any one has idea how to do it 
thanks in advance

**Comment by zoran995** [comment]
Hi @toufa123
Terria currently has a minimap that is available only in Cesium viewer (as part of the pedestrian tool). It currently doesn't support the leaflet but it can be used as a base for building a general version that will work everywhere (I think that it isn't on the team radar for development).

also, I have a private version of minimap feature built for a client but there is currently no plan to open it to public

**Reply by nerminealayeb7** [reply]
@zoran995  how can I show the minimap within the view, please?
